tests/components/ozw/test_light.py

```diff
@@ -149,3 +149,335 @@ async def test_light(hass, light_data, light_msg, sent_messages):
+    assert msg["payload"] == {"Value": 255, "ValueIDKey": 659128337}
+
+    # Feedback on state
+    light_msg.decode()
+    light_msg.payload["Value"] = byte_to_zwave_brightness(255)
+    light_msg.encode()
+    light_rgb_msg.decode()
+    light_rgb_msg.payload["Value"] = "#0000ff0000"
+    light_rgb_msg.encode()
+    receive_message(light_msg)
+    receive_message(light_rgb_msg)
+    await hass.async_block_till_done()
+
+    state = hass.states.get("light.led_bulb_6_multi_colour_level")
+    assert state is not None
+    assert state.state == "on"
+    assert state.attributes["rgb_color"] == (0, 0, 255)
+
+    # Test setting hs_color
+    new_color = [300, 70]
+    await hass.services.async_call(
+        "light",
+        "turn_on",
+        {"entity_id": "light.led_bulb_6_multi_colour_level", "hs_color": new_color},
+        blocking=True,
+    )
+    assert len(sent_messages) == 11
+    msg = sent_messages[-1]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": 255, "ValueIDKey": 659128337}
+
+    msg = sent_messages[-2]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": "#ff4cff0000", "ValueIDKey": 659341335}
+
+    # Feedback on state
+    light_msg.decode()
+    light_msg.payload["Value"] = byte_to_zwave_brightness(255)
+    light_msg.encode()
+    light_rgb_msg.decode()
+    light_rgb_msg.payload["Value"] = "#ff4cff0000"
+    light_rgb_msg.encode()
+    receive_message(light_msg)
+    receive_message(light_rgb_msg)
+    await hass.async_block_till_done()
+
+    state = hass.states.get("light.led_bulb_6_multi_colour_level")
+    assert state is not None
+    assert state.state == "on"
+    assert state.attributes["hs_color"] == (300.0, 70.196)
+
+    # Test setting rgb_color
+    new_color = [255, 154, 0]
+    await hass.services.async_call(
+        "light",
+        "turn_on",
+        {"entity_id": "light.led_bulb_6_multi_colour_level", "rgb_color": new_color},
+        blocking=True,
+    )
+    assert len(sent_messages) == 13
+    msg = sent_messages[-1]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": 255, "ValueIDKey": 659128337}
+
+    msg = sent_messages[-2]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": "#ff99000000", "ValueIDKey": 659341335}
+
+    # Feedback on state
+    light_msg.decode()
+    light_msg.payload["Value"] = byte_to_zwave_brightness(255)
+    light_msg.encode()
+    light_rgb_msg.decode()
+    light_rgb_msg.payload["Value"] = "#ff99000000"
+    light_rgb_msg.encode()
+    receive_message(light_msg)
+    receive_message(light_rgb_msg)
+    await hass.async_block_till_done()
+
+    state = hass.states.get("light.led_bulb_6_multi_colour_level")
+    assert state is not None
+    assert state.state == "on"
+    assert state.attributes["rgb_color"] == (255, 153, 0)
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5844602)